bin/generators/credentials/list.js

```diff
@@ -6,33 +6,45 @@ module.exports = class extends eg.Generator {
 
     this.configureCommand({
       command: 'list [options]',
-      description: 'List all credentials for Consumer (User or App)',
+      description: 'List all active credentials for Consumer (User or App)',
       builder: yargs =>
         yargs
           .usage(`Usage: $0 ${process.argv[2]} list [options]`)
           .example(`$0 ${process.argv[2]} list -c 7498d1a9-7f90-4438-a9b7-0ba4c6022353`)
           .string(['c'])
+          .boolean(['a', 'd'])
           .describe('c', 'Consumer ID: can be User ID or username or app ID')
+          .describe('a', 'List activated and deactivated credentials')
+          .describe('d', 'List only deactivated credentials')
           .alias('c', 'consumerId').nargs('c', 1)
-          .group(['c'], 'Options:')
+          .alias('a', 'all')
+          .alias('d', 'deactivated')
+          .group(['c', 'a'], 'Options:')
     });
   }
 
   prompting () {
-    return this.admin.credentials.list(this.argv.consumerId)
+    const {consumerId, all, deactivated} = this.argv;
+    let mode = 'activeOnly';
```

Comment:
this will be sent using url, so it is better to have to have lower case params. 

I suggest:
active
disabled or archived